docs/src/reference/custom.md

```diff
@@ -0,0 +1,24 @@
+# CusBono IntBound Problem 
+
+This section contain information about custom_bonobo.jl , integer_bounds.jl and problem.jl
```

Comment:
This section contains the customization of the branch-and-bound implementation of Bonobo.jl.

We don't need the subsection here.